lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -219,6 +205,15 @@
         }
       } // EditorHas.functionBar
 
+      if (ActiveOptions['section'] && $.markupSupportsEditableSections ( ActiveOptions.MarkupType )) {
+        var sectionLine = $.findSection(ActiveOptions['section'], LanguageDefinition.getDefinitionFor('gollum-helpers'));
+        if (sectionLine) {
+          window.ace_editor.gotoLine(sectionLine+1, offset=0, slap=false); //gotoLine starts counting at 1 instead of 0, so +1
```

Comment:
Please replace these dummy variable names with meaningful ones! 😂 
```javascript
window.ace_editor.gotoLine(sectionLine+1, column=0, animate=false);
```

Reply:
Hahaha more coffee.

Reply:
Do you think it would make more sense to place the cursor at the end of the header line?